ci/docker/Dockerfile.build.ubuntu

```diff
@@ -83,9 +82,31 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         autoconf \
         gperf \
         libb2-dev \
-        libzstd-dev && \
+        libzstd-dev \
+        gfortran && \
     rm -rf /var/lib/apt/lists/*
 
+# Build OpenBLAS from source
+RUN export LIBRARY_PATH=$LIBRARY_PATH:/usr/lib/gcc/x86_64-linux-gnu/7/ && \
```

Comment:
Is the library path needed for clang?

Reply:
Yeah other wise it would not find gfortran